src/os/FileStore.cc

```diff
@@ -579,29 +579,29 @@ FileStore::FileStore(const std::string &base, const std::string &jdev, osflagbit
   PerfCountersBuilder plb(g_ceph_context, internal_name, l_os_first, l_os_last);
 
-  plb.add_u64(l_os_jq_max_ops, "journal_queue_max_ops");
-  plb.add_u64(l_os_jq_ops, "journal_queue_ops");
-  plb.add_u64_counter(l_os_j_ops, "journal_ops");
-  plb.add_u64(l_os_jq_max_bytes, "journal_queue_max_bytes");
-  plb.add_u64(l_os_jq_bytes, "journal_queue_bytes");
-  plb.add_u64_counter(l_os_j_bytes, "journal_bytes");
-  plb.add_time_avg(l_os_j_lat, "journal_latency");
-  plb.add_u64_counter(l_os_j_wr, "journal_wr");
-  plb.add_u64_avg(l_os_j_wr_bytes, "journal_wr_bytes");
-  plb.add_u64(l_os_oq_max_ops, "op_queue_max_ops");
-  plb.add_u64(l_os_oq_ops, "op_queue_ops");
-  plb.add_u64_counter(l_os_ops, "ops");
-  plb.add_u64(l_os_oq_max_bytes, "op_queue_max_bytes");
-  plb.add_u64(l_os_oq_bytes, "op_queue_bytes");
-  plb.add_u64_counter(l_os_bytes, "bytes");
-  plb.add_time_avg(l_os_apply_lat, "apply_latency");
-  plb.add_u64(l_os_committing, "committing");
-
-  plb.add_u64_counter(l_os_commit, "commitcycle");
-  plb.add_time_avg(l_os_commit_len, "commitcycle_interval");
-  plb.add_time_avg(l_os_commit_lat, "commitcycle_latency");
-  plb.add_u64_counter(l_os_j_full, "journal_full");
-  plb.add_time_avg(l_os_queue_lat, "queue_transaction_latency_avg");
+  plb.add_u64(l_os_jq_max_ops, "journal_queue_max_ops", "Max operations in throttle queue");
+  plb.add_u64(l_os_jq_ops, "journal_queue_ops", "Operations in throttle queue");
+  plb.add_u64_counter(l_os_j_ops, "journal_ops", "Total operations in throttle");
+  plb.add_u64(l_os_jq_max_bytes, "journal_queue_max_bytes", "Max size of throttle queue");
+  plb.add_u64(l_os_jq_bytes, "journal_queue_bytes", "Size of throttle queue");
+  plb.add_u64_counter(l_os_j_bytes, "journal_bytes", "Total operations size in throttle");
+  plb.add_time_avg(l_os_j_lat, "journal_latency", "Average throttle queue completing latency");
+  plb.add_u64_counter(l_os_j_wr, "journal_wr", "Actually written operations");
+  plb.add_u64_avg(l_os_j_wr_bytes, "journal_wr_bytes", "Actually written size");
+  plb.add_u64(l_os_oq_max_ops, "op_queue_max_ops", "Max operations in queue");
+  plb.add_u64(l_os_oq_ops, "op_queue_ops", "Operations in queue");
+  plb.add_u64_counter(l_os_ops, "ops", "Total proceeded operations");
+  plb.add_u64(l_os_oq_max_bytes, "op_queue_max_bytes", "Max size of queue");
+  plb.add_u64(l_os_oq_bytes, "op_queue_bytes", "Size of queue");
+  plb.add_u64_counter(l_os_bytes, "bytes", "Total size of work");
+  plb.add_time_avg(l_os_apply_lat, "apply_latency", "Average latency of operation execution");
+  plb.add_u64(l_os_committing, "committing", "Is committing something just now");
+
+  plb.add_u64_counter(l_os_commit, "commitcycle", "Commit cycles");
+  plb.add_time_avg(l_os_commit_len, "commitcycle_interval", "Average interval between commits");
+  plb.add_time_avg(l_os_commit_lat, "commitcycle_latency", "Average latency of commit");
+  plb.add_u64_counter(l_os_j_full, "journal_full", "Journal full states");
```

Comment:
"Journal writes while full" would be clearer -- "states" implies that we are somehow experiencing an increasing number of different states, rather than the number of times we have seen one particular state (fullness)